src/OpenTelemetry.Resources.Process/ProcessResourceBuilderExtensions.cs

```diff
@@ -17,8 +17,19 @@ public static class ProcessResourceBuilderExtensions
     /// <param name="builder">The <see cref="ResourceBuilder"/> being configured.</param>
     /// <returns>The instance of <see cref="ResourceBuilder"/> being configured.</returns>
     public static ResourceBuilder AddProcessDetector(this ResourceBuilder builder)
+    {
+        return AddProcessDetector(builder, true);
```

Comment:
nit: Slightly better readability IMO:

```suggestion
        return AddProcessDetector(builder, includeProcessOwner: true);
```